go.mod

```
@@ -217,3 +217,5 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	tags.cncf.io/container-device-interface/specs-go v0.6.0 // indirect
 )
+
+replace github.com/hashicorp/go-retryablehttp v0.7.5 => github.com/suse/go-retryablehttp v0.0.0-20241209123412-5c0e967751af
```

Is it possible to publish a tag from your `suse-v0.7.5` branch on the fork? It should make it easier to pinpoint the ref in future.

---

Better yet, how about cutting a release?

---

And what should the release number be? 0.7.5.1? That's a bit excessive tbh and golang doesn't handle four digit releases very well

---

How about tagging `v0.7.5-suse`?

```suggestion
replace github.com/hashicorp/go-retryablehttp v0.7.5 => github.com/suse/go-retryablehttp v0.7.5-suse
```

---

And then when we push another fix to the branch, we do what?

---

We bump it? `v0.7.6-suse`, which shouldn't confuse readers with the upstream releases given the obvious `-suse` suffix.

---

Upstream already released 0.7.6 and our backport would have no connection to that release.

I get your gripe with using the hash, which annoys me as well, because my initial `replace` has the branch name instead of this hash.

---

This particular case isn't really representative of things going bad if we tend to use a particular tag/branch given the project's relative inactivity, but I agree with you, let's stick to the commit hash for this one.

Is it possible to add a comment block at least? a simple comment and a permalink to the commit would be helpful imo.